features-json/wasm-bigint.json

```diff
@@ -0,0 +1,603 @@
+{
+  "title":"WebAssembly Non-trapping float-to-int Conversion",
```

Comment:
@dschuff I noticed this is the wrong title, have replaced it with "WebAssembly BigInt to i64 conversion in JS API"

Reply:
Ah thanks. I'm sure I just forgot to update it when copy/pasting.